packages/neon-core/src/rpc/clients/RpcDispatcher.ts

```diff
@@ -49,6 +49,9 @@ export class RpcDispatcher {
   public url: string;
 
   public constructor(url: string) {
+    if (!url.startsWith("http")) {
+      throw new Error("Please provide an url that starts with http or https");
```

Comment:
Overall PR looks good to me 👍

Just an open question here to understand a bit of your design philosophy; any specific reason you've not chosen to be slightly more strict here? As in something like
```javascript
if (!url.startsWith("http://") && !url.startsWith("https://")) {
```
I can imagine a `http:/` (single `/`) is easy to make.

Reply:
The fix was merely to guard against people forgetting to append the `http` scheme for the url. Solving for confusion (people are used to `localhost` urls working without the http scheme attached) vs typos (`http:/` is a clear visible typo).

Reply:
I'm getting strange behaviour though if I make the typo
```
const rpc_client = new Neon.rpc.RPCClient("http:/127.0.0.1:10332");
rpc_client.getBlockCount().catch((reason) => {
  console.log(reason);
});
```
Note how the port has changed to `:80` just because I forgot a `/`. That would throw me off wondering where the address came from.
```
Error: connect ECONNREFUSED 127.0.0.1:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1145:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 80,
  config: {
    url: 'http:/127.0.0.1:10332',
    method: 'post',
```

Reply:
actually it is worse if you happen to not use `127.0.0.1`, try something like `http:/1.2.3.4`.